Device.js

```diff
@@ -1,4 +1,4 @@
-var { NativeModules } = require('react-native');
+var { NativeModules, NativeAppEventEmitter } = require('react-native');
```

Comment:
'destructuring expression' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'require' is not defined.